### PR TITLE
docs: update branding

### PR DESCRIPTION
**Pull request by mozhoku**
## Description

This PR aims to enhance the documentation by explicitly stating that this repository is a **fork** of Tier IV's AWSIM repository. Naming it **AWSIM Labs**.
 
### Related links

- https://github.com/autowarefoundation/AWSIM/issues/35

## Tests performed

No tests performed.

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by xmfcx** [comment]
~For some reason I was not able to push to your branch.~

~Could you fast-forward this to https://github.com/autowarefoundation/AWSIM/tree/docs/update-branding~

It works now.

**Comment by xmfcx** [comment]
Could you update the readme as well?

Could you also create a markdown table with feature differences from the upstream AWSIM?

**Comment by mozhoku** [comment]
@xmfcx I've added table and updated the readme. Please let me know if you want any other changes.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
-# AWSIM
+# AWSIM Labs
 
-![](/README_img/AWSIM.png)
+![](docs/assets/images/E2ESim.png)
 
-AWSIM is the best scene simulator for [Autoware](https://github.com/autowarefoundation/autoware).
+![](docs/assets/images/autoware-foundation.png){: style="height:90px"}
+![](docs/assets/images/awsim-labs-logo.png){: style="height:90px"}
```
Comment (xmfcx) [inline README.md]: This looks like this:
Screenshot from 2024-04-17 14-50-36

See: https://github.com/autowarefoundation/AWSIM/blob/e7054c0fb7a693b9a8e9975331330d7b6904cb7e/README.md

File: README.md [hunk 2]
```diff
@@ -11,31 +16,34 @@ AWSIM is the best scene simulator for [Autoware](https://github.com/autowarefoun
...
 - Made with Unity Game Engine
+- Multiple scene and vehicle setup
+- Interactable simulation and UI
 
 ## Tutorial
 
-First, try the tutorial !  
-[AWSIM Document - Quick Start Demo](https://tier4.github.io/AWSIM/GettingStarted/QuickStartDemo/)
+First, try the tutorial!  
+[AWSIM Document - Quick Start Demo](https://autowarefoundation.github.io/AWSIM/main/GettingStarted/QuickStartDemo/)
```
Comment (xmfcx) [inline README.md]:
```suggestion
[AWSIM Labs Documentation - Quick Start Demo](https://autowarefoundation.github.io/AWSIM/main/GettingStarted/QuickStartDemo/)
```

File: README.md [hunk 3]
```diff
@@ -11,31 +16,34 @@ AWSIM is the best scene simulator for [Autoware](https://github.com/autowarefoun
...
-Everyone is welcome !
-1. Create a derived branch `feature/***` from the `main` branch.
-2. Create a pull request for the `main` branch.
+Everyone is welcome!
+1. Create a issue [here](https://github.com/autowarefoundation/AWSIM/issues) to discuss the contribution you want to make.
+2. Create a derived branch `feature/***` from the `main` branch.
+3. Create a pull request for the `main` branch.
 
-see also [AWSIM document - Git branch](https://tier4.github.io/AWSIM/ProjectGuide/GitBranch/)
+see also [AWSIM document - Git branch](https://autowarefoundation.github.io/AWSIM/main/ProjectGuide/GitBranch/)
```
Comment (xmfcx) [inline README.md]:
```suggestion
see also [AWSIM Labs Documentation - Git branch](https://autowarefoundation.github.io/AWSIM/main/ProjectGuide/GitBranch/)
```

**Review by xmfcx — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mozhoku) [reply README.md]: Okay, fixed it :+1:

File: README.md [hunk 4]
```diff
@@ -1,8 +1,12 @@
-# AWSIM
+# AWSIM Labs
 
-![](/README_img/AWSIM.png)
+![](docs/assets/images/E2ESim.png)
```
Comment (xmfcx) [inline README.md]:
```suggestion
![](docs/assets/images/E2ESim.png)
```

could you reduce the max size of this too?

**Comment by xmfcx** [comment]
brave_EOtTN5nbsq

Please update these to AWSIM Labs too.

File: README.md (hunk 4 above)
Comment (mozhoku) [reply README.md]: Do you have a specific value for max size in your mind or should I set it whatever value I have in my mind?

File: mkdocs.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-site_name: AWSIM document
+site_name: AWSIM Labs document
```
Comment (xmfcx) [inline mkdocs.yml]:
```suggestion
site_name: AWSIM Labs Documentation
```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,12 @@
-# AWSIM
+# AWSIM Labs
 
-![](/README_img/AWSIM.png)
+<img src="docs/assets/images/E2ESim.png" width="640" height="360">
```
Comment (xmfcx) [inline README.md]:
```suggestion
<img src="docs/assets/images/E2ESim.png" height="200">
```

**Comment by xmfcx** [comment]
Can you remove https://github.com/autowarefoundation/AWSIM/tree/main/README_img too since now it is unused?

File: README.md (hunk 6 above)
Comment (mozhoku) [reply README.md]: Done in: https://github.com/autowarefoundation/AWSIM/pull/42/commits/e40ca28813972a7f960d640e20f062376f1ac62b

**Review by xmfcx — APPROVED** [review]

**Pull request merged**